integration/resources/templates/combination/graphqlapi_pipeline_resolver.yaml

```diff
@@ -86,13 +86,14 @@ Resources:
         }
       Auth:
         Type: API_KEY
-      DynamoDBDataSources:
-        PostsDataSource:
-          TableName: !Ref DynamoDBPostsTable
-          TableArn: !GetAtt DynamoDBPostsTable.Arn
-        PostsLogDataSource:
-          TableName: !Ref DynamoDBPostsLogTable
-          TableArn: !GetAtt DynamoDBPostsLogTable.Arn
+      DataSources:
+        DynamoDb:
```

Comment:
We should take a more thorough look at capitalization overall, but I'd lean toward `DynamoDB` here. `DynamoDb` would make sense with other text following it, but by itself it seems to lower legibility. And I'd assume customers would instinctively type `DynamoDB`, instead of having to remember it's `DynamoDb`, when it's already inconsistent (while there's good reason for it) with things such as `GraphQLApi`. So all that to say I'm not sure, but I think this deserves more thought.

Reply:
After some offline discussion, we are going to hold an internal UX review on how we name our SAM properties since we have some inconsistencies throughout the package. We will provide updates once we reach a conclusion.